dataprofiler/labelers/column_name_model.py

```diff
@@ -65,10 +67,25 @@ def _validate_parameters(self, parameters):
             "false_positive_dict",
             "include_label",
             "negative_threshold_config",
+            "positive_threshold_config",
         ]
 
         list_of_accepted_parameters = optional_parameters + required_parameters
 
+        if parameters["true_positive_dict"]:
+            label_map_dict_keys = set(self.label_mapping.keys())
+            true_positive_unique_labels = set(
+                parameters["true_positive_dict"][0].values()
+            )
+
+            # if not a subset that is less than or equal to
+            # label mapping dict
+            if true_positive_unique_labels > label_map_dict_keys:
+                errors.append(
+                    """`true_positive_dict` must be a subset
+                        of the `label_mapping` values()"""
+                )
+
```

Comment:
checking that `true_positive_dict` label values are together an equal or lesser subset of `label_mapping`

Reply:
Ideally we would say what is not supposed to be there, but that can always be done later.